.github/workflows/build.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           - "7.3"
           - "7.4"
```

Comment:
It seems it's 7.4+ now.

Reply:
Appears to be driven from here: https://github.com/yiisoft/yii2-symfonymailer/blob/abd79bd4de49e962c5b3ed2b4c0799e2b73e478a/composer.json#L28

Tests pass once composer can install packages successfully.

Reply:
Yes, we cannot use it like that. I would add it as a suggestion (`suggest` section).

Reply:
My thinking was that since the template is for new projects, and PHP's official guidance is that <7.4 is EOL ( https://www.php.net/supported-versions.php ), the probability of someone cloning this onto an old version to start a project was relatively low. 

Or at least tests don't need to be maintained for < 7.4 since this doesn't update like a normal vendor package, since it is expected to modify the composer.json file and adopt it as the file for the project. Also, at that point, someone could disable this package if it really needs to work on < 7.4, and we can include verbiage in the readme perhaps...?

I'm dealing with the opposite issue essentially - deprecated package warnings get us dinged on security audits...